src/transfer/messages.rs

```diff
@@ -28,7 +28,7 @@ pub struct SubmitRequest {
 
 #[derive(Serialize, Deserialize, Debug)]
 pub struct CancelRequest {
-    pub job_id: JobId,
+    pub job_ids: Vec<JobId>,
```

Comment:
Please use `JobSelector` here (see `JobInfoRequest`). A function to convert `JobSpecifier` to `JobSelector` will also probably be needed. We can also rename `JobSpecifier` to `JobSelectorArg` or something like that.